spec/index-spec.ts

```diff
@@ -32,4 +32,8 @@ describe('index', () => {
     expect(index.noop).to.exist;
     expect(index.identity).to.exist;
   });
+
+  it('should export constants', () => {
+    expect(index.EMPTY).to.exist;
```

Comment:
what's expected usecases of exported `EMPTY` constant? (in other words, end user how do they use it?)

Reply:
Basically, users should use `EMPTY` instead of `empty()`... mostly because it gives them the idea that all the empties they're using are the same instance.

Reply:
not needed in this specific PR, but feeling it's worth to mention in somewhere in docs later.